### DISCUSSION TITLE
Labelling z-stack images

### DISCUSSION THREAD

**Discussion by MartinTheuerkauf** [wkentaro/labelme#1093] [Ideas / Feature requests]
I would like to label microscopic images that come as z-stack of several focus levels. 

LabelMe does open such images but only shows the first layer. Is there any possibility to browse the other layers as well? Or could such option be added?

**Comment by amirh132** [comment]
Was a solution found?

**Reply by MartinTheuerkauf** [reply]
We produces a browser based solution, should be upload soon.
